packages/auth/src/AuthProvider.tsx

```diff
@@ -26,6 +26,11 @@ export interface AuthContextInterface {
    * but does not update the current user state.
    **/
   getCurrentUser(): Promise<null | CurrentUser>
+  /**
+   * Checks if the "currentUser" from the api side
+   * is assigned a role
+   **/
+  hasRole(): Promise<boolean>
```

Comment:
```suggestion
  hasRole(): boolean
```
 Since it's checking against the data that's already in `currentUser` this probably doesn't have to be a promise

Reply:
Ah yes. An earlier implementation has `hasRole` calling `getCurrentUser` hence the async but then realized did not need to do that.